daemon/container_config.h

```diff
@@ -1,6 +1,6 @@
 /*
  * This file is part of trust|me
- * Copyright(c) 2013 - 2017 Fraunhofer AISEC
+ * Copyright(c) 2013 - 2019 Fraunhofer AISEC
```

Comment:
2020

Reply:
he commit is from 2019 i will add a separate patch for this